node/config/utils.go

```diff
@@ -26,3 +26,7 @@ func (c *BatcherNodeConfig) GetShardsIDs() []types.ShardID {
 func (rc *RouterNodeConfig) GetMaxSizeBytes() (uint64, error) {
 	return rc.RequestMaxBytes, nil
 }
+
+func (rc *RouterNodeConfig) ToValidateSignature() (bool, error) {
+	return rc.ValidateSignature, nil
+}
```

Comment:
The norm in Go is to call a getter just by the name of the var; i.e.

`func ( rc *RouterNodeConfig) ValidateSignature() bool`

also
`func (rc *RouterNodeConfig) MaxSizeBytes() uint64`
remove the error, it is redundent

Reply:
PS; should it not be `RequestMaxBytes` for consistency with the config yaml?